src/ts/utils/SubtitleUtils.ts

```diff
@@ -87,6 +87,8 @@ export class SubtitleSwitchHandler {
     if (this.listElement.hasItem(subtitle.id)) {
       this.listElement.removeItem(subtitle.id);
     }
+
+    this.selectCurrentSubtitle();
```

Comment:
This is a minor fix which is needed since no select boxes are used anymore (the problem exists on v2 as well).

In case the active SubtitleTrack gets removed we need to update to the now selected subtitle track (usually `off`)